src/views/OpenOrders.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-title>10 of 26 orders</ion-title>
+        <ion-buttons  slot="end">
+        <ion-menu-button>
+          <ion-icon :icon="options" />
+        </ion-menu-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+    
+    <ion-content :fullscreen="true">
+      <ion-searchbar /> 
+
+      <div class="filters">
+        <ion-row class="filter-scroll">
+          <ion-item lines="none">
+            <ion-checkbox slot="start"/>
+            <ion-label>
+              <h5>Same day</h5>
```

Comment:
just use ion label here. why are you using h5?

Reply:
Removed sir